source/GameGraphRoot.cpp

```diff
@@ -25,7 +25,7 @@ constexpr float PI_180 = (float)(M_PI / 180);
 constexpr float BREACH_SCALE = 0.25;
 
 /** The scale of the donut textures. */
-constexpr float DONUT_SCALE = 0.5;
+constexpr float DONUT_SCALE = 0.32;
 
```

Comment:
Shouldn't there be an `f` at the end of the float, ie `0.32f`?

Reply:
Fixed

Reply:
Wait, it's not tho...